packages/appstore/src/components/account/account.tsx

```diff
@@ -1,56 +1,276 @@
+                            <AccountDisplay
+                                is_eu={is_eu}
+                                market_type={market_type}
+                                server={server}
+                                sub_account_type={sub_account_type}
+                                has_error={has_error}
+                                platform={platform}
+                                is_dark_mode_on={is_dark_mode_on}
+                                shortcode={shortcode}
+                                should_show_server_name={should_show_server_name}
+                            />
+                        )}
                     </Text>
                     <Text className='account__container--account-details-wrapper--name-number--number'>
-                        {account_number}
+                        {loginid_text}
                     </Text>
                 </div>
 
                 <Text className='account__container--account-details-wrapper--balance'>
-                    {account_balance}
-                    {` ${getCurrencyDisplayCode(currency)}`}
+                    {`${balance} ${getCurrencyDisplayCode(currency)}`}
                 </Text>
             </div>
-            <div className='account__container--account-reset-button'>
-                <Text className='account__container--account-reset-button--label'>
-                    <Localize i18n_default_text={account_button} />
-                </Text>
-            </div>
-            <div className='account__container--dropdown'>
-                <WalletIcon icon={'DropDown'} />
+            <div className='account__container--account-reset-button-wrapper'>
+                {has_reset_balance ? (
+                    <Button
+                        is_disabled={is_disabled}
+                        onClick={(e: React.ChangeEvent<HTMLInputElement>) => {
+                            e.stopPropagation();
+                            onClickResetVirtualBalance();
+                        }}
+                        className='acc-switcher__reset-account-btn'
+                        secondary
+                        small
+                    >
+                        {localize('Reset')}
+                    </Button>
+                ) : (
+                    <Button
+                        is_disabled={is_disabled}
+                        onClick={onClickDeposit}
+                        className='acc-switcher__reset-account-btn'
+                        secondary
+                        small
+                    >
+                        {localize('Deposit')}
+                    </Button>
+                )}
             </div>
+            {isMobile() && getCurrencyDisplayCode(currency) !== 'Demo' && (
+                <div className='account__container--dropdown'>
+                    <WalletIcon icon={'DropDown'} />
+                </div>
+            )}
         </div>
     );
 };
 
 export default OptionsAccount;
```

Comment:
move it to the end of the file

Reply:
I don't understand the suggestion.

Reply:
you are exporting the component in the middle of the file, move it to the last line of the code